packages/bsync/src/routes/put-operation.ts

```diff
@@ -82,6 +82,17 @@ const validateOp = (req: PutOperationRequest): Operation => {
     throw new ConnectError('operation method is invalid', Code.InvalidArgument)
   }
 
+  if (req.method === Method.CREATE || req.method === Method.UPDATE) {
+    try {
+      JSON.parse(new TextDecoder().decode(req.payload))
```

Comment:
I think elsewhere I saw `.toString('utf8')` used.  Would that work just fine?

Reply:
Ah, I had assumed that `req.payload` was a `Buffer`.  I see it's a `Uint8Array`.  You'll find we use the `uint8arrays` library pretty liberally for working with those, or on the server they are easy to wrap in a `Buffer`.